### PR TITLE
Merge upstream master, upgrade OSCAL 1.0.0

### PR DESCRIPTION
**Pull request by danielnaab**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,17 +1,4 @@
 [submodule "oscal"]
 	path = oscal
 	url = https://github.com/usnistgov/OSCAL.git
-	branch = master
```
Comment (ohsh6o) [inline .gitmodules:4]: B: My bad, can you restore this?

**Review by ohsh6o — CHANGES_REQUESTED** [review]

**Review by ohsh6o — CHANGES_REQUESTED** [review]
This is looking great! Can I make one more request? I just [compared against upstream](https://github.com/GSA/fedramp-automation/compare/master...18F:upgrade-oscal-1.0.0) and only 91 files changed (and only the things we care about. Nice work!

Can you please update for the one last change I made to the `fedramp1.0.0-oscal1.0.0` release tag (there is a commit I pushed to the release after we paired) and fix relevant tests? Thanks!

**Comment by danielnaab** [comment]
> Can you please update for the one last change I made to the `fedramp1.0.0-oscal1.0.0` release tag (there is a commit I pushed to the release after we paired) and fix relevant tests? Thanks!

Sure - would it be possible to merge that to `GSA/master` first? (for sanity, because this branch includes a merge from master)

**Comment by ohsh6o** [comment]
> Sure - would it be possible to merge that to `GSA/master` first? (for sanity, because this branch includes a merge from master)

OK, fine by me.

**Review by ohsh6o — APPROVED** [review]

**Comment by GaryGapinski** [comment]
Validations work against the `fedramp1.0.0-oscal1.0.0` version.

I found and corrected some Schematron tests while reviewing the validations. I'll commit the corrections later this morning.

**Pull request merged**